app/models/planning_application.rb

```diff
@@ -279,13 +279,31 @@ def result_present?
   end
 
   def validation_requests
-    (description_change_validation_requests + replacement_document_validation_requests + additional_document_validation_requests + other_change_validation_requests + red_line_boundary_change_validation_requests).sort_by(&:created_at).reverse
+    (replacement_document_validation_requests + additional_document_validation_requests + other_change_validation_requests + red_line_boundary_change_validation_requests).sort_by(&:created_at).reverse
   end
 
   def cancelled_validation_requests
     validation_requests.filter(&:cancelled?).sort_by(&:cancelled_at).reverse
   end
 
+  def open_description_change_requests
+    description_change_validation_requests.open
+  end
+
+  def responded_description_request
+    description_change_validation_requests.closed.select do |request|
+      request.rejection_reason != "Request cancelled by planning officer." && request.auto_closed.blank?
```

Comment:
why do you need this check `request.rejection_reason != "Request cancelled by planning officer."` ?
Can the request be cancelled another way ? If not then you can just check if `cancelled_at` is set.
And if `auto_closed` is a boolean why are we using `blank?` here ? Should it not be false

Then could we not do something like  `scope :responded, -> { closed.where(cancelled_at: nil, auto_closed: false) }` 

Then call it with `description_change_validation_requests.responded.last`

Reply:
Maybe for all these `last` calls we need to be sorting by order(created_at: :asc)

Reply:
so this was before we had a "cancelled state" I was just checking the rejection reason since the state could be "closed" instead of cancelled - you're right, now that we have cancelled I can just check that instead, thanks!